### ISSUE TITLE
Does this really work???

### ISSUE THREAD

**Issue by garymcintire** [andrewliao11/dni.pytorch#4]
Looking at the code here, I do not see the synthetic gradients module.
dni_linear seems to just be a regular MLP with batch normalization
Am I missing something???

**Comment by andrewliao11** [comment]
Hi @garymcintire , 
In fact, my ```dni_linear``` is the synthetic gradient module. The main network will be  updated based on the output of the ```dni_linear```, instead of back-propagation signal from loss function.
please see [L37 in train.py](https://github.com/andrewliao11/dni.pytorch/blob/master/train.py#L37).

**Comment by andrewliao11** [comment]
I'll first close this issue.
feel free to reopen it if there are other questions

**Comment by garymcintire** [comment]
Thanks for clearing this up, Andrew!

-----Original Message-----
